bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -9,317 +9,230 @@ metadata:
-                terminationGracePeriodSeconds: 10
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: search-v2-operator-controller-manager
+              terminationGracePeriodSeconds: 10
       permissions:
-        - rules:
-            - apiGroups:
-                - apps
-              resources:
-                - deployments
-                - replicasets
-              verbs:
-                - '*'
-            - apiGroups:
-                - ""
-              resources:
-                - configmaps
-                - secrets
-                - serviceaccounts
-                - services
-                - persistentvolumeclaims
-                - pods
-                - endpoints
-              verbs:
-                - '*'
-            - apiGroups:
-                - coordination.k8s.io
-              resources:
-                - leases
-              verbs:
-                - get
-                - list
-                - watch
-                - create
-                - update
-                - patch
-                - delete
-            - apiGroups:
-                - ""
-              resources:
-                - events
-              verbs:
-                - create
-                - patch
-          serviceAccountName: search-v2-operator
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: search-v2-operator-controller-manager
     strategy: deployment
   installModes:
-    - supported: true
-      type: OwnNamespace
-    - supported: false
-      type: SingleNamespace
-    - supported: false
-      type: MultiNamespace
-    - supported: true
-      type: AllNamespaces
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
   keywords:
-    - Advanced Cluster Management
-    - acm
-    - Open Cluster Management
-    - ocm
-    - odyssey
-    - search
-    - stolostron
+  - search
+  - Advanced Cluster Management
+  - Open Cluster Management
+  - stolostron
+  - acm
+  - ocm
   links:
-    - name: Search Operator
-      url: https://github.com/stolostron/search-v2-operator
+  - name: Search V2 Operator
+    url: https://search-v2-operator.domain
```

Comment:
We should keep the url? We can update the search-v2-operator/config/manifests/bases/search-v2-operator.clusterserviceversion.yaml so that it reflects here.

Reply:
changed in `search-v2-operator/config/manifests/bases/search-v2-operator.clusterserviceversion.yaml`